docs/datasets/danish.md

```diff
@@ -354,6 +354,41 @@ $ scandeval --model <model-id> --dataset scandiqa-da
 ```
 
 
+### Belebele
+
+This dataset was published in [this paper](https://aclanthology.org/2024.acl-long.44/) and is a large-scale multilingual reading comprehension dataset covering 122 languages. The questions are generated from Wikipedia articles and are designed to test various aspects of reading comprehension, including factual understanding, inference, and numerical reasoning.
+
+The dataset provides training, validation, and test splits with human-verified question-answer pairs. The questions are generated to be answerable from the given context and cover diverse topics from Wikipedia articles.
+
+When evaluating generative models, we use the following setup (see the [methodology](/methodology) for more information on how these are used):
+
+- Number of few-shot examples: 4
+- Prefix prompt:
+  ```
+  Nedenfor følger tekster med tilhørende spørgsmål og svar.
+  ```
+- Base prompt template:
+  ```
+  Tekst: {text}
+  Spørgsmål: {question}
+  Svar: {label}
+  ```
+- Instruction-tuned prompt template:
+  ```
+  Tekst: {text}
+
+  Besvar følgende spørgsmål om teksten ovenfor.
+
+  Spørgsmål: {question}
+  ```
```

Comment:
These are not quite right - they're specified in the `src/scripts/create_belebele.py` script and/or the appropriate dataset config in the `src/scandeval/dataset_configs.py` module.